implementations/rust/ockam/ockam_vault_aws/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+//! AWS implementation of the ockam_vault::Kms trait
+//!
+#![deny(unsafe_code)]
+#![warn(
+    missing_docs,
+    trivial_casts,
+    trivial_numeric_casts,
+    unused_import_braces,
+    unused_qualifications
+)]
+#![cfg_attr(not(feature = "std"), no_std)]
+
```

Comment:
extra newline might be confusing

Reply:
I think that's ok, the attribute applies to the whole module, not to the next declaration. It is consistent with similar declarations in other `lib` files